homeassistant/components/huisbaasje/config_flow.py

```diff
@@ -0,0 +1,84 @@
+"""Config flow for Huisbaasje integration."""
+import logging
+
+from huisbaasje import Huisbaasje, HuisbaasjeConnectionException, HuisbaasjeException
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_ID, CONF_PASSWORD, CONF_USERNAME
+from homeassistant.data_entry_flow import AbortFlow
+
+from .const import DOMAIN  # pylint: disable=unused-import
+
+_LOGGER = logging.getLogger(__name__)
+
+DATA_SCHEMA = vol.Schema(
+    {vol.Required(CONF_USERNAME): str, vol.Required(CONF_PASSWORD): str}
+)
+
+
+class HuisbaasjeConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Huisbaasje."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_CLOUD_POLL
+
+    async def async_step_user(self, user_input=None):
+        """Handle a flow initiated by the user."""
+        if user_input is None:
+            return await self._show_setup_form(user_input)
+
+        errors = {}
+
+        try:
+            user_id = await self._validate_input(user_input)
+
+            _LOGGER.info("Input for Huisbaasje is valid!")
+
+            # Set user id as unique id
+            await self.async_set_unique_id(user_id)
```

Comment:
Please only wrap the line that can raise in the try..
except block. Use an `else` block if needed.